Assets/MixedRealityToolkit.ThirdParty/MRTK-Quest/Scripts/Config/MRTKOculusConfig.cs

```diff
@@ -106,10 +108,20 @@ public static MRTKOculusConfig Instance
         [Tooltip("Custom hand material to use for hand tracking hand mesh.")]
         private Material customHandMaterial = null;
 
+        public System.Action OnCustomHandMaterialUpdate;
```

Comment:
Please add a new line between OnCustomHandMaterialUpdate and the custom hand material property

Reply:
Also please add a summary block for the action, as it is public

Reply:
Done